src/feature/flow_chart_panel/components/node-edit-menu/ParamField.tsx

```diff
@@ -23,13 +22,14 @@ const ParamField = ({
   const { updateCtrlInputDataForNode } = useFlowChartState();
   const handleChange = (value: string) => {
     updateCtrlInputDataForNode(nodeId, paramId, {
-      functionName,
+      functionName: functionName,
       param: paramId,
-      value,
+      value: value,
```

Comment:
Same here, for ES6 we don't need to set value explicitly if the key and value are the same.